Dockerfile.openhabian-aarch64-native

```diff
@@ -9,7 +9,7 @@ ENV INITSYSTEM on
 ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
 
 RUN apt-get update -qq \
-    && apt-get install --yes -qq --no-install-recommends systemd systemd-sysv git wget python3 python3-pip apt-utils jq virt-what \
+    && apt-get install --yes -qq --no-install-recommends systemd systemd-sysv git wget python3 python3-pip apt-utils jq virt-what lsb-release unzip psmisc \
```

Comment:
Maybe create a separate Dockerfile for BATS, else we'll forget why this line is different across Dockerfiles.

Reply:
Added a comment block, IMHO better than creating another config.
Maybe we should silently extend the installed packages also in all other config files.